doc/mgr/zabbix.rst

```diff
@@ -113,6 +114,14 @@ This can be done with this command:
 
 The module will now send its latest data to the Zabbix server.
 
+Items discovery is accomplished also via sabbix_sender, and runs every `discovery_interval * interval` seconds. If you wish to launch discovery 
```

Comment:
Type in *zabbix_sender* here

Reply:
Typo fixed.

Actually, it handles big amount of data pretty well. We've tested it not with ceph (as we don't have so big cluster), but we currently have a couple of hosts with about 6000 discovered items and it works pretty well with them. But trapper has a limit of 128Mb of data per one connection, but I can recall, that since some zabbix version (like 3.0) zabbix_sender is able to split incoming data into separate packages, so that shouldn't be a problem either.

Reply:
I see. I haven't tested this either nor do I have a setup to properly test this on.

Can you share some screenshots of Zabbix where it has discovered the items?